assets/src/edit-story/utils/test/useRovingTabIndex.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Internal dependencies
+ */
+import { getSiblingDirection } from '../useRovingTabIndex';
+
+describe('getSiblingDirection', () => {
+  describe('RTL', () => {
+    it.each(['ArrowRight', 'ArrowUp', 'PageUp'])(
+      'given key: %s returns "previousSibling"',
+      (key) => {
+        const isRTL = true;
```

Comment:
nit: the `isRTL` constant can be pulled out of each of these tests and placed in the describe block instead 🤷 not a big deal